src/cascadia/TerminalApp/ColorScheme.cpp

```diff
@@ -120,36 +131,60 @@ void ColorScheme::LayerJson(const Json::Value& json)
     }
 }
 
-winrt::hstring ColorScheme::Name() const noexcept
-{
-    return _schemeName;
-}
-
-winrt::com_array<Color> ColorScheme::Table() const noexcept
+// Method Description:
+// - Create a new serialized JsonObject from an instance of this class
+// Arguments:
+// - scheme: a ColorScheme object that will be converted into a serialized JsonObject
+// Return Value:
+// - a serialized JsonObject from the values in scheme
+Json::Value ColorScheme::ToJson(const TerminalApp::ColorScheme& scheme)
 {
-    winrt::com_array<Color> result{ COLOR_TABLE_SIZE };
-    std::transform(_table.begin(), _table.end(), result.begin(), [](til::color c) -> Color { return c; });
-    return result;
+    Json::Value json{ Json::ValueType::objectValue };
+    const auto schemeImpl{ winrt::get_self<implementation::ColorScheme>(scheme) };
+    schemeImpl->UpdateJson(json);

```

Comment:
is there a reason we should have a static `ToJson` instead of just a member method? I know you're following the layout for FromJson/LayerJson, but that exists _because there isn't an object yet when you call FromJson_

Reply:
I guess a followup question is: is there ever a reason to call `UpdateJson` with the same json value multiple times to layer-IN multiple values? I'm worried that there is not.

Reply:
I just did it to match FromJson/LayerJson. But good point, we don't need it.